Library/Homebrew/cmd/update-bash.sh

```diff
@@ -208,6 +208,12 @@ update-bash() {
     esac
   done
 
+  if ! curl -I --connect-timeout 2 https://github.com &> /dev/null
```

Comment:
Oh I was forgetting to tell it to die with `-t` - That's why it was hanging rather than exiting on failure last night.

`ping` is actually a bit slower than either of the header checks, a whole second even on success:
```
ping -c5 -t1 8.8.8.8  0.00s user 0.00s system 0% cpu 1.005 total
```
If you're not worried about that though I have no objections, although Xu's comment about firewalls could be an issue.